openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -266,6 +266,26 @@ Workspace ForwardTranslator::translateModelPrivate( model::Model & model, bool f
     }
   }
 
+  // Energyplus only allows single zone input for ITE object. If space type is assigned in OS, 
+  // will translate to multiple ITE objects assigned to each zone under the same space type.
+  // then delete the one that pointed to a spacetype.
+  // By doing this, we can solve the potential problem that if this load is applied to a space type, 
+  // the load gets copied to each space of the space type, which may cause conflict of supply air node.
+  std::vector<ElectricEquipmentITEAirCooled> iTEAirCooledEquipments = model.getConcreteModelObjects<ElectricEquipmentITEAirCooled>();
```

Comment:
Not really. I've tried using the existing code but it doesn't include what this part does. CombineSpace will merge ITE object of each space under the same **ThermalZone** into single ITE object, while this part distribute ITE object associated with **SpaceType** to each space under it, which will generate multiple ITE objects. They function differently.

It is mainly because ITE object can be assigned to a spacetype in OpenStudio but can't be assigned to a zonelist in EnergyPlus.